feat: FIT-493: Make left column of User Assigner infinitely paginated and searchable

yyassi-heartex opened the pull request:
This pull request makes a minor adjustment to the `Select` component in `select.tsx` to standardize the height of the dropdown list. Instead of dynamically sizing the dropdown based on the number of available items, it now always renders space for five items.

* UI consistency: The dropdown list in the `Select` component now always renders space for five items, regardless of the number of available options. This change improves consistency in the user interface.

yyassi-heartex committed:
feat: FIT-493: Make left column of User Assigner infinitely paginated and searchable

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-fit-493/infinite-members

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-fit-493/infinite-members

borisheartex commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16933827651)

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16933827651

borisheartex committed:
Merge branch 'develop' into 'fb-fit-493/infinite-members'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16933827651
